README.md

```diff
@@ -36,8 +36,26 @@ repos.
 
 ## Usage
 
-TODO: fill in details and examples here.
+### Commitlint
 
+* Add a file called commitlint.config.js to your project root with the following content:
+
+```javascript
+'use strict';
+
+module.exports = {
+   extends: [ '@silvermine/eslint-config/commitlint.js' ],
+};
+```
+
+* Use git log --oneline to find the short hash of the previous commit and take note of it
+* Add the following NPM script to `package.json`:
+
+   `"commitlint": "commitlint --from deadbeef"` (where deadbeef is the short hash from the previous step)
+
+* Configure `.gitlab-ci.yml` or `.travis.yml` to run `npm run commitlint` before running `npm test`
+
+TODO: fill in more details and examples here
```

Comment:
We can probably remove this line now too.

Reply:
ADIRL we'll leave this for now since technically there's still some docs TODO :D